shared/test/test_net_sim_api.rb

```diff
@@ -15,7 +15,7 @@ def setup
     @table_name = 'n' # for "node table"
 
     # Every test should start with an empty table
```

Comment:
There should be a test that verifies that the net_sim_api interacts correctly with the pubsub service (i.e. calls publish when things change.)